### ISSUE TITLE
atom.xml is not found (link from default theme)

### ISSUE THREAD

**Issue by cfjedimaster** [hexojs/hexo#2413]
I made a brand new site just to test things out - didn't change anything (except the author in _config.yml), and I noticed the RSS link from the default template leads to a 404 at atom.xml.

## Environment Info

Node version(`node -v`)
```
v6.9.2.
```

Your site `_config.yml`
```
# Hexo Configuration
## Docs: https://hexo.io/docs/configuration.html
## Source: https://github.com/hexojs/hexo/

# Site
title: Hexo
subtitle:
description:
author: Raymond Camden
language:
timezone:

# URL
## If your site is put in a subdirectory, set url as 'http://yoursite.com/child' and root as '/child/'
url: http://yoursite.com
root: /
permalink: :year/:month/:day/:title/
permalink_defaults:

# Directory
source_dir: source
public_dir: public
tag_dir: tags
archive_dir: archives
category_dir: categories
code_dir: downloads/code
i18n_dir: :lang
skip_render:

# Writing
new_post_name: :title.md # File name of new posts
default_layout: post
titlecase: false # Transform title into titlecase
external_link: true # Open external links in new tab
filename_case: 0
render_drafts: false
post_asset_folder: false
relative_link: false
future: true
highlight:
  enable: true
  line_number: true
  auto_detect: false
  tab_replace:

# Category & Tag
default_category: uncategorized
category_map:
tag_map:

# Date / Time format
## Hexo uses Moment.js to parse and display date
## You can customize the date format as defined in
## http://momentjs.com/docs/#/displaying/format/
date_format: YYYY-MM-DD
time_format: HH:mm:ss

# Pagination
## Set per_page to 0 to disable pagination
per_page: 10
pagination_dir: page

# Extensions
## Plugins: https://hexo.io/plugins/
## Themes: https://hexo.io/themes/
theme: landscape

# Deployment
## Docs: https://hexo.io/docs/deployment.html
deploy:
  type:

...
```

Your theme `_config.yml`
```
landscape
...
```

Plugin version(`npm ls --depth 0`)
```
hexo-site@0.0.0 c:\Users\ray\Desktop\trash\hexo1
+-- hexo@3.2.2
+-- hexo-generator-archive@0.1.4
+-- hexo-generator-category@0.1.3
+-- hexo-generator-index@0.2.0
+-- hexo-generator-tag@0.2.0
+-- hexo-renderer-ejs@0.2.0
+-- hexo-renderer-marked@0.2.11
+-- hexo-renderer-stylus@0.3.1
`-- hexo-server@0.2.0

```

## For BUG

Make a new Hexo site, run it, and click on the RSS icon in the upper right hand corer.

**Comment by NoahDragon** [comment]
The hexo-generate-feed has to be installed. https://github.com/hexojs/hexo-generator-feed

**Comment by cfjedimaster** [comment]
Fair enough - but if this is the out of the box theme, should it be included, or if not, should the link be removed?

My point is - for what you see when you first learn Hexo, it appears broken. There is a good reason for it to be broken, but it's a bad initial user experience.

**Comment by NoahDragon** [comment]
You are right, it is not user-friendly. I think it is caused by migrating Hexo from 2.0 to 3.0. Before 3.0, all the plugins were included in Hexo, but in 3.0, the plugins are added based on usage. We try to keep the core Hexo as minimum as possible, all extended functionalities could be accessed via plugins.

I will add a note in Landscape theme to notify users.
